README.md

```diff
@@ -319,6 +252,98 @@ Example of a partial CR, where the second service refers to a Secret containing
 
 -->
 
+<!--
+  ## Minimum requirements for being bindable
```

Comment:
When we say minimum requirements for being bindable, does it mean that these fields are `required`. IMO some fields are `optional` like `dataMapping` and `detectBindingResources`.